page.codeberg.maerchenfeeimgarten.Pfeilspiel.yaml

```diff
@@ -0,0 +1,25 @@
+id: page.codeberg.maerchenfeeimgarten.Pfeilspiel
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: "Pfeilspiel_TNG_international"
+modules:
+  - name: Pfeilspiel_TNG_international
+    buildsystem: simple
+    build-commands:
+      - install -D  Pfeilspiel_TNG_international /app/bin/Pfeilspiel_TNG_international
+      - install -Dm 0644 page.codeberg.maerchenfeeimgarten.Pfeilspiel.desktop /app/share/applications//page.codeberg.maerchenfeeimgarten.Pfeilspiel.desktop
+      - install -Dm 0644 Pfeilspiel_TNG.svg /app/share/icons/hicolor/scalable/apps/page.codeberg.maerchenfeeimgarten.Pfeilspiel.svg
+    sources:
+      - type: file
+        sha256: 79c40f98189e4ffcf91f0390218e04aedd744be767d833ec5c7aabb64e7586c2
+        url: https://codeberg.org/MaerchenfeeimGarten/Pfeilspiel/releases/download/v0.0.8/Pfeilspiel_TNG_international
```

Comment:
so how will this work on multiple architectures?

Reply:
No, it will only work on amd64. Is it enought to add `only-arches: [x86_64]` to this source or do I have to do more?

Reply:
OK, I think this should be enough.

Reply:
there is still an aarch64 package built.

Reply:
> OK, I think this should be enough.

https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on

Reply:
Thank you for this link. I didn't know I have to read the maintenance section if I want to publish the very first package version...